src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -3,24 +3,40 @@
             [puppetlabs.trapperkeeper.authorization.rules :as rules]
             [puppetlabs.trapperkeeper.authorization.ring :as ring]
             [puppetlabs.ssl-utils.core :as ssl-utils])
-  (:import  (java.net InetAddress)
-            (clojure.lang IFn)))
+  (:import  (clojure.lang IFn)))
 
+(def is-authentic-key
```

Comment:
DRY'ed up in ac142dd..ab2be69, now located in ring.clj since these constants seem most related to the ring request itself rather than the rules.